R/return_hidden_items.R

```diff
@@ -5,7 +5,7 @@
 #'
 #' @param x a \code{secuTrialdata} object
 #' @export
-#' @details return_hidden_items will produce a data.frame that pinpoints which items are hidden.
+#' @return a data.frame that pinpoints which items are hidden
```

Comment:
maybe useful to list the columns?

Reply:
Columns? I'm confused.

Reply:
each column/variable in the data.frame

Reply:
But maybe rather in the details section? Explaining each of the columns of the returned data frame in the @return is a bit uncommon.

Reply:
return is the appropriate place for it in my opinion. There are packages that also do it (most packages aren't fully documented like this though, stopping at eg "secuTrialdata object", which is why it's uncommon)